docs/docs/policy-reference/_examples/rego/example/policy.rego

```diff
@@ -0,0 +1,21 @@
+package example
```

Comment:
I think a better path for this example is 

```
docs/docs/policy-reference/_examples/rego/rule_metadata
```

or something like that? just in case we add more in future.